src/functionalTest/java/uk/gov/hmcts/reform/pbis/utils/DeadLetterQueueHelper.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.pbis.utils;
+
+import com.microsoft.azure.servicebus.ClientFactory;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+
+public class DeadLetterQueueHelper implements AutoCloseable {
+
+    private static final Logger logger = LoggerFactory.getLogger(DeadLetterQueueHelper.class);
+    private final IMessageReceiver messageReceiver;
+    private final Duration maxReceiveWaitTime;
+
+    public DeadLetterQueueHelper(
+        String namespaceConnectionString,
+        String topicName,
+        String subscriptionName,
+        Duration maxReceiveWaitTime) throws ServiceBusException, InterruptedException {
+
+        this.messageReceiver = buildMessageReceiver(
+            namespaceConnectionString,
+            topicName,
+            subscriptionName
+        );
+
+        this.maxReceiveWaitTime = maxReceiveWaitTime;
+    }
+
+    public IMessage receiveMessage() throws ServiceBusException, InterruptedException {
+        logger.info("Receiving message...");
```

Comment:
This could be raised to debug/trace level as something will be logged later on.

Reply:
As just discussed, I considered debug, but came to the conclusion that it would be better to always have this information in the output of integration tests. This way when they fail, we can easily find out what happened.